mcp_server.py

```
@@ -591,6 +795,636 @@ def calculate_detailed_metrics(tree: ast.AST, source_code: str, include_halstead
+        return [TextContent(type="text", text=f"❌ Erro ao parar auto-documentação: {str(e)}")]
+
+
+async def handle_get_docs_status(arguments: Dict[str, Any]) -> List[TextContent]:
+    """Handle getting documentation and automation status."""
+    global server_state
+    
+    try:
+        # Get various status information
+        watcher_stats = auto_docs_watcher.get_statistics()
+        git_status = git_manager.get_git_status()
+        doc_stats = doc_generator.get_generation_stats()
```

Ruff (0.11.9)

Undefined name `doc_generator`

(F821)